library/public/interactor.h

```diff
@@ -252,6 +252,10 @@ class F3D_EXPORT interactor
   virtual interactor& startAnimation() = 0;
   virtual interactor& stopAnimation() = 0;
   [[nodiscard]] virtual bool isPlayingAnimation() = 0;
+  /**
+   * Access animation data.
+   */
+  [[nodiscard]] virtual std::string getAnimationName() = 0;
```

Comment:
looking back at the current implementation we can see that there is a:

`scene::availableAnimations` method.

I wonder if it should be better to get all available animation names with a 

`scene::availableAnimationNames` and then use the animation_indices option to recover the one that are enabled.

Btw such discussion may be better on discord :)

https://discord.f3d.app

Reply:
I agree, but lets discuss on discord for faster turnaround.